core-bundle/src/EventListener/DataContainer/LayoutOptionsListener.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener\DataContainer;
+
+use Contao\CoreBundle\ServiceAnnotation\Callback;
+use Doctrine\DBAL\Connection;
+
+/**
+ * @Callback(table="tl_page", target="fields.layout.options")
+ * @Callback(table="tl_page", target="fields.subpagesLayout.options")
+ */
+class LayoutOptionsListener
```

Comment:
This should implent `ResetInterface` (and be tagged accordingly) to reset the layouts on kernel shutdown.

Reply:
Good point, implemented in https://github.com/contao/contao/pull/3419/commits/ad9dbb73cba25a9576274e0d4a1eadbe43797e19.